lib/facter/systemd.rb

```diff
@@ -1,17 +1,17 @@
 # Fact: systemd
 #
-# Purpose: 
-#   Determine whether systemd is the init system on the node
+# Purpose:
+#   Determine whether SystemD is the init system on the node
```

Comment:
it's plain systemd, not SystemD according to https://www.freedesktop.org/wiki/Software/systemd/

Reply:
oh dang. I did some copy and paste issues. my patch for this file isn't based on master. I will fix that

Reply:
fixed